js/app.js

```diff
@@ -2,241 +2,193 @@
-  {
+  }
+
+  CookieStore.prototype.cookieSales = function () {
     for(var i = 0; i <storeHours.length; i++ ){
       var numberSold = this.customerPerHour[i] * this.averageCookiesPerCustomer;
       this.cookiesPerHour[i] = numberSold;
       this.dailySales += numberSold;
-    }
-  },
-  renderSales: function (){
-    var ulElem = document.getElementById('seaTacAir');
-    for (var i = 0; i < storeHours.length; i++){
-      //create an element
-      var listItemElem = document.createElement( 'li' );
-      //Give it a content
-      listItemElem.textContent = storeHours[i] + ': ' + Math.round(this.cookiesPerHour[i]) + ' cookies';
-      //Append to the DOM
-      ulElem.appendChild(listItemElem);
+      cookiesPerHour.push(this.numberSold);
+      totalCookieSales.push(dailySales);
 
-    }
-    listItemElem.textContent = 'Total of cookies: ' + Math.round(this.dailySales) + ' cookies.';
-  }
 
-};
 
-seaTac.randomNumberOfClient();
-seaTac.cookieSalesPerHour();
-seaTac.renderSales();
-console.log(seaTac.dailySales);
+      CookieStore.prototype.sales = function () {
+        return 'Hi Pat, today\'s sales at' + this.store + ' were: ' + this.cookieSales + '.';
+      };
 
-//Seattle Center LOCATION
+    }
 
-var seattleCenter = {
-  minClientPerHour: 11,
-  maxClientPerHour: 38,
-  averageCookiesPerCustomer: 3.7,
-  dailySales:0,
-  cookiesPerHour:[],
-  customerPerHour:[],
+  };
+};
 
-  randomNumberOfClient: function ()//calculates the random clients
-  {
-    for (var i = 0; i < storeHours.length; i++){
+var firstandPike = new CookieStore ('First and Pike', 23, 65,6.3);
+console.log(firstandPike);
+var seaTacAirport = new CookieStore ('SeaTac Airport', 3, 24, 1.2);
+console.log(seaTacAirport);
+var seattleCenter = new CookieStore ('Seattle Center', 11,38,3.7);
+console.log(seattleCenter);
+var alki = new CookieStore ('Alki',2,16,4.6);
+console.log(alki);
+var capitolHill = new CookieStore ('Capitol Hill', 20,38,2.3);
+console.log(capitolHill);
 
-      this.customerPerHour[i] = Math.floor(Math.random() * (this.maxClientPerHour - this.minClientPerHour)) + this.minClientPerHour;
-    }
-  },
-  cookieSalesPerHour: function () //calculates sales per hour and total sales
-  {
-    for(var i = 0; i <storeHours.length; i++ ){
-      var numberSold = this.customerPerHour[i] * this.averageCookiesPerCustomer;
-      this.cookiesPerHour[i] = numberSold;
-      this.dailySales += numberSold;
-    }
-  },
-  renderSales: function (){
-    var ulElem = document.getElementById('seattleCen');
-    for (var i = 0; i < storeHours.length; i++){
-      //create an element
-      var listItemElem = document.createElement( 'li' );
-      //Give it a content
-      listItemElem.textContent = storeHours[i] + ': ' + Math.round(this.cookiesPerHour[i]) + ' cookies.';
-      //Append to the DOM
-      ulElem.appendChild(listItemElem);
+//console.log(firstandPike.sales());
+CookieStore.firstandPike();
```

Comment:
It looks like you're expecting to have a `sales()` method on CookieStore, but the only methods you've created are `cookieSales` and `randomNumberOfClient`.